src/multi_vector_simulator/server.py

```diff
@@ -107,13 +118,33 @@ def run_simulation(json_dict, epa_format=True, **kwargs):
     logging.debug("Accessing script: B0_data_input_json")
     dict_values = B0.convert_from_json_to_special_types(json_dict)
 
+    # if True will return the lp file's content in dict_values
+    lp_file_output = dict_values[SIMULATION_SETTINGS][OUTPUT_LP_FILE][VALUE]
+    # to avoid the lp file being saved somewhere on the server
+    dict_values[SIMULATION_SETTINGS][OUTPUT_LP_FILE][VALUE] = False
```

Comment:
I don´t understand: 

Whether or not the LP file is stored on the server already is decided in `run_oemof`: https://github.com/rl-institut/multi-vector-simulator/blob/250192a3b65b66154047d5b56d82a0dd9bb51f43/src/multi_vector_simulator/D0_modelling_and_optimization.py#L108
and `store_lp_file()`: https://github.com/rl-institut/multi-vector-simulator/blob/250192a3b65b66154047d5b56d82a0dd9bb51f43/src/multi_vector_simulator/D0_modelling_and_optimization.py#L250

Is it necessary to move writing the LP file out here, and giving `local_energy_system` back to `run_simulation()`?

Reply:
I did this to avoid having to modify MVS and introduce a way for MVS to know if it is run in server mode or in local mode. Maybe this would be the way to go though...

Reply:
It only modifies the existing code if `return_les` is `True` and it is `False` by default. Thus the giving of `local_energy_system` is only done in explicit cases (where the user explicitely set `return_les` to `True`). I thought it was a safe way of modifying the code minimally to still be able to let the user download the LP file from the server